opendbc/car/honda/carcontroller.py

```diff
@@ -150,6 +151,7 @@ def update(self, CC, CS, now_nanos):
     # steer torque is converted back to CAN reference (positive when steering right)
     apply_torque = int(np.interp(-limited_torque * self.params.STEER_MAX,
                                  self.params.STEER_LOOKUP_BP, self.params.STEER_LOOKUP_V))
+    self.steer_restricted = (abs(apply_torque) == self.params.STEER_MAX) # for hud lane line display
```

Comment:
Let's not arbitrarily define what already exists https://github.com/commaai/opendbc/blob/7e5239c747a0b47e2ba3e6ad1aa10acff14bfe38/opendbc/car/car.capnp#L419